vnext/Directory.Build.props

```diff
@@ -17,7 +17,7 @@
       The PR (windows-vs-pr.yml) and CI (publish.yml() turn it back on.
     -->
     <EnableSourceLink Condition="'$(EnableSourceLink)' == ''">false</EnableSourceLink>
-    <FollyVersion>2020.09.14.00</FollyVersion>
+    <FollyVersion>2020.10.05.00</FollyVersion>
```

Comment:
Woudl you mind chcking the failure in the publish step? It has been failing to copy the files in Layout-Headers.ps1 to inc\folly which makes packing up nuget files fail. I see you paraemeterized it there to read this file, but when looking in the build artifact created, the files are not there...